specs/merge/beacon-chain.md

```diff
@@ -66,6 +69,19 @@ We define the following Python custom types for type hinting and readability:
 
 ## Containers
 
+### Modified containers
+
+#### `Eth1Data`
+
+*Note*: The only modification is the type of `block_hash` field that is changed to `Hash32`.
```

Comment:
I think it's fine to change the phase0 spec and update the type. It's just a change in presentation.

Reply:
agreed

Reply:
moved to a phase 0 type mod